pkg/cli/admin/release/mirror.go

```diff
@@ -307,27 +322,64 @@ func (o *MirrorOptions) Run() error {
 	}
 	version = is.Name
 
+	// sourceFn is given a chance to rewrite source mappings
+	sourceFn := func(ref imagesource.TypedImageReference) imagesource.TypedImageReference {
+		return ref
+	}
 	var mappings []mirror.Mapping
-	if len(o.From) > 0 && !o.SkipRelease {
+	if len(o.From) > 0 {
 		src := o.From
-		srcRef, err := imagereference.Parse(src)
+		srcRef, err := imagesource.ParseReference(src)
```

Comment:
This will also allow `s3://` reference, not sure if you want to allow that?

Reply:
I think we do at some point, just haven't gotten a chance to get back to it.  We're keeping the s3 in our pocket so we can mirror to static stuff like file.  In theory s3 and file should always work everywhere.